nexml_nyiso/notebooks/model_scoring.py

```diff
@@ -1,6 +1,71 @@
 import utils
 import pandas as pd
+from bokeh.plotting import figure
+from bokeh.models import DatetimeTickFormatter
 
 
-def eval():
-    pass
+def de_process(df, mean, std):
+    for col in utils.COLUMNS_TO_NORMALIZE:
+        if col in list(df.columns):
+            df[col] *= std[col]
+            df[col] += mean[col]
+
+
+def eval(model, mean, std):
+    """
+    Evaluates trained model.
+
+    Parameters
+    ----------
+    model: Model -> Trained Keras model.
+    mean: Series -> Series containing mean of columns.
+    std: Series -> Series containing std of columns.
+
+    Returns DataFrame and Bokeh plot for notebook use.
+    """
+    train, test = utils.load_data()
```

Comment:
Fitting a scalar to the entire dataset would cause data leakage. From what I've read, it is generally considered best practice to fit on the training data only so the test data remains completely unseen.